src/BasketBundle/Block/BasketBlockService.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Xavier Coureau <xcoureau@ekino.com>
  */
-class BasketBlockService extends BaseBlockService
+class BasketBlockService extends AbstractBlockService
```

Comment:
I am not sure if it was an error but take a look: https://github.com/sonata-project/SonataBlockBundle/blob/3.x/Block/BaseBlockService.php